src/algebra/ordered_ring.lean

```diff
@@ -403,9 +403,6 @@ end
 lemma exists_le_mul_self (a : α) : ∃ x : α, a ≤ x * x :=
 let ⟨x, hx⟩ := exists_lt_mul_self a in ⟨x, le_of_lt hx⟩
 
-/- TODO This theorem ought to be written in the context of `nontrivial` linearly ordered (additive)
-commutative monoids rather than linearly ordered rings; however, the former concept does not
-currently exist in mathlib. -/
```

Comment:
Why the deletion?

Reply:
See [this discussion](https://github.com/leanprover-community/mathlib/pull/4762#discussion_r511497222).